pkg/testutil/nerdtest/requirements.go

```diff
@@ -171,6 +171,20 @@ var CGroup = &test.Requirement{
 	},
 }
 
+var RootfulOrCGroupV2 = &test.Requirement{
```

Comment:
```suggestion
var CgroupAccessible = &test.Requirement{
```

Reply:
Better. Thanks.

Reply:
The commit message and the PR description have to be updated too

Reply:
Done.